### ISSUE TITLE
webcam

### ISSUE THREAD

**Issue by pradeepsharma07** [AISangam/Facenet-Real-time-face-recognition-using-deep-learning-Tensorflow#12]
How to use this for webcam for live face recognition?

**Comment by Aashima-Gupta** [comment]
yes, pls someone guide us. How to use this for real time face recognition on windows?

**Comment by pradeepsharma07** [comment]
Hi ,
I did it on real time face recognition using webcam.

**Comment by srijansingh53** [comment]
In the file identify_face_video.py, line 50/51 
change the statement
video_capture = cv2.VideoCapture(input_video) to
video_capture = cv2.VideoCapture(0)

NB: 0 in VideoCapture(0) is the webcam device no. you are assigning to...

**Comment by helloimchristine** [comment]
@Pradeep07shar can you provide your code? TQ VERY MUCH.

**Comment by helloimchristine** [comment]
@srijansingh53 do you have the source code? I'm really interested in this

**Comment by shankarvishnu** [comment]
You have to update [this line](https://github.com/shankarvishnu/Facenet-Real-time-face-recognition-using-deep-learning-Tensorflow/blob/b351d5d3f4766572868236eed78a25082da44c27/identify_face_video.py#L50) to use webcam.
video_capture = cv2.VideoCapture(input_video)
to 
video_capture = cv2.VideoCapture(0)
